### PR TITLE
Coconut 022

### PR DESCRIPTION
**Pull request by yvendruscolo**
Hello! Thank you for your contribution to `polyglot-euler`.
- [✅] I checked [CONTRIBUTING.md](https://github.com/FrankKair/polyglot-euler/blob/master/CONTRIBUTING.md) for my programming language of choice.

### How the solution works

* `parse` does just that: reads the file, splits into names and sorts it

* `score` calculates the score of a single name, according to the rules

* `names` "indexes" the iterator, by zipping it to a `range`, then multiplies the tuple (index, score), and sums the whole thing

### Performance

using `$ time coconut -n -r p022.coco`

```
real    0m2.162s
user    0m2.085s
sys     0m0.071s
```

### CODE REVIEW

File: src/022/p022.coco [hunk 1]
```diff
@@ -0,0 +1,7 @@
+parse = open ..> .read() ..> .split(",") ..> sorted
+
+score = .[1:-1] ..> map$(ord ..> (+)$(-64)) ..> reduce$(+)
+
+names = zip$(range(1, 6000)) ..> starmap$(*) ..> reduce$(+)
+
+"p022_names.txt" |> parse |> map$(score) |> names |> print
```
Comment (FrankKair) [inline src/022/p022.coco]: Maybe you could wrap this last call in a solve function and call it afterwards. What do you think?

Comment (yvendruscolo) [reply src/022/p022.coco]: Thy wish is my command (once i get home)

Comment (yvendruscolo) [reply src/022/p022.coco]: Also, how can i test these file dependent problems on tio.run?

Comment (FrankKair) [reply src/022/p022.coco]: I don't think you can, since there's no way to upload a file to the environment (as far as I know).

**Pull request merged**